app/views/sign_up/registrations/_required_pii_accordion.html.erb

```diff
@@ -56,9 +56,9 @@
     </span>
   </p>
   <p class="grid-row grid-gutter">
-    <%= image_tag(
-          asset_url('p-key.svg'),
-          size: '40', alt: t('devise.registrations.start.bullet_6_img'), class: 'margin-top-05 grid-col-auto',
+    <%= render AlertIconComponent.new(
+          icon_name: :personal_key,
+          size: '40', alt: t('devise.registrations.start.bullet_6_img'), class: 'margin-top-05 grid-col-auto'
```

Comment:
Based on the failing build, you'll either need to remove, update, or account for the `size` attribute here, since it's incompatible with the `AlertIconComponent`'s own default `width` and `height`.

```suggestion
          width: 40,
          height: 40,
          alt: t('devise.registrations.start.bullet_6_img'),
          class: 'margin-top-05 grid-col-auto',
```

Or something like this in `AlertIconComponent`:

```rb
def size_attributes
  return tag_options[:size] if tag_options[:size].present?
  { width: DEFAULT_WIDTH, height: DEFAULT_HEIGHT }
end
```